jarbas/core/management/commands/companies.py

```diff
@@ -1,6 +1,6 @@
 import csv
 import lzma
-
```

Comment:
This line separating native modules imports and third-party module imports should be kept (as it's a PEP8 recommendation)